fnet/src/vespa/fnet/transport_thread.h

```diff
@@ -156,6 +154,9 @@ class FNET_TransportThread : public FastOS_Runnable
      **/
     bool InitEventLoop();
 
+    void endEventLoop();
+    void checkTimedoutComponents(vespalib::duration timeout);
```

Comment:
should not be public

Reply:
It is in the private: section so I think this is good.

Reply:
this should not be together with the event loop external API, then

Reply:
my bad, I forgot we removed the single-threaded option a long time ago